providers/coordinator.go

```diff
@@ -46,11 +46,16 @@ var BuiltinCoreID = coreconf.Config.ID
 
 var Coordinator ProvidersCoordinator
 
-func newCoordinator() *coordinator {
+func newCoordinator(globalAutoUpdateCfg UpdateProvidersConfig) *coordinator {
+	log.Info().
```

Comment:
Info level logs are always displayed to the user. Can we switch these new log messages to Debug level?